modules/recommended-configurable-storage-technology.adoc

```diff
@@ -93,22 +93,20 @@ In a scaled/HA {product-title} registry cluster deployment:
 * All NAS storage are not recommended for {product-title} Registry cluster
 deployment with production workloads.
 
-
 === Metrics
 
 In an {product-title} hosted metrics cluster deployment:
 
-* The preferred storage technology is block storage.
-* It is not recommended to use NAS storage for a hosted metrics cluster deployment with production workloads.
+* The preferred storage technology is block storage. Do not use file storage for
+metrics.
 
 [IMPORTANT]
 ====
-Testing shows issues with using the NFS server on RHEL as storage backend for
-the container image registry. This includes the Cassandra for metrics storage.
-Therefore, using NFS to back PVs used by core services is not recommended.
+Testing shows significant unrecoverable corruptions using file storage and,
+therefore, is not recommended for use.
 
-Other NFS implementations on the marketplace might not have these issues.
-Contact the individual NFS implementation vendor for more information on any
+There are file storage implementations in the marketplace might not have these
```

Comment:
s/might/that might 
Or: There might be file storage implementation in the marketplace that do not have these issues.